libs/src/utils/utils.js

```diff
@@ -1,10 +1,17 @@
 const bs58 = require('bs58')
 const Web3 = require('../web3')
 const axios = require('axios')
+const Hashids = require('hashids/cjs')
 
 const MultiProvider = require('./multiProvider.js')
 const uuid = require('./uuid.js')
 
+// Hashids
+
+const HASH_SALT = 'azowernasdfoia'
```

Comment:
ive actually been using this tool a bunch, nice to see it here

Reply:
yeah we have this code duplicated in client, CN, and identity - libs is clearly where it belongs heh